glide.yaml

```diff
@@ -83,4 +83,6 @@ import:
 - package: github.com/google/go-github/github
   version: v30.1.0
 - package: github.com/golang/mock
-  version: v1.4.3
\ No newline at end of file
+  version: v1.4.3
+- package: github.com/hashicorp/go-version
```

Comment:
It seems you have added a new library but it's not in the vendor directory. Thus the builds are failing.

Reply:
yes, i will add do that